### PR TITLE
Easi 2573/possible operational solution treat as custom

### PR DESCRIPTION
**Pull request by StevenWadeOddball**
# [EASI-2573](https://jiraent.cms.gov/browse/EASI-2573), [EASI-2502](https://jiraent.cms.gov/browse/EASI-2502), [EASI-2752](https://jiraent.cms.gov/browse/EASI-2752)

## Changes and Description

- 2573
    - Update Operational Solutions to have the ability to treat certain defined options as Other (eg Contractor)
    - Updated Possible Operational Solutions to set the correct ones as treat as Other
    - Added isOther and otherHeader to OperationalSolution
        - Custom solutions and defined treat as other solutions return isOther true
    - Removed addOrUpdate solution
       - Discrete mutation to insert
       - Discrete mutation to update
       - Remove legacy  update / insert mutation
  
- 2502
   - Update Operational Solutions to match the options in FIGMA and google sheets
   -  Updated Operational Solution links to match what is provided in the spreadsheet.
   
   
  Front End
   - Changes made in https://github.com/CMSgov/mint-app/pull/532
   

## How to test this change
Backend
1. Use the updated mutations in the postman collection to verify the new resolvers
2. Verify that the need solution links match google sheets / figma

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

File: pkg/graph/schema.graphql [hunk 1]
```diff
@@ -1289,7 +1289,10 @@ type OperationalSolution {
     pocEmail: String
     mustStartDts: Time
     mustFinishDts: Time
+    isOther: Boolean
```
Comment (ClayBenson94) [inline pkg/graph/schema.graphql:1292]: Should this be `Boolean!`? Looks like both Operational Solutions and Possible Operational Solutions have a `isOther`

File: pkg/graph/schema.graphql [hunk 2]
```diff
@@ -1303,10 +1306,14 @@ type OperationalSolution {
 
 input OperationalSolutionChanges @goModel(model: "map[string]interface{}"){
     needed: Boolean
+    name: String
```
Comment (ClayBenson94) [inline pkg/graph/schema.graphql:1309]: Can you remind me when `name` needs to get passed? (Perhaps it could use a comment like the one for `nameOther`)

Comment (StevenWadeOddball) [reply pkg/graph/schema.graphql:1309]: Good call Clay! I think you are right, that this shouldn't be in the changes object. Name should be coming from possible operational solution, I will update this!

**Review by ClayBenson94 — CHANGES_REQUESTED** [review]

File: pkg/graph/schema.graphql (hunk 1 above)
Comment (StevenWadeOddball) [reply pkg/graph/schema.graphql]: Good call! Thanks Clay! I updated this.

File: pkg/models/possible_operational_solution.go
[the remark this answers is not in the document]
Comment (StevenWadeOddball) [reply pkg/models/possible_operational_solution.go:1]: Good catch! Model and queries updated to reflect this

**Review by ClayBenson94 — APPROVED** [review]

**Pull request merged**